connectors/src/connectors/intercom/temporal/workflows.ts

```diff
@@ -52,6 +55,11 @@ export async function intercomSyncWorkflow({
 }) {
   await saveIntercomConnectorStartSync({ connectorId });
 
+  // Add folder node for teams
+  await upsertIntercomTeamsFolderActivity({
+    connectorId,
+  });
+
```

Comment:
we're in for a ride again @philipperolet :)

Reply:
intercom is webhook-based for the conversations

Reply:
oh wait actually we're good, I thiiink webhooks rely on non-temporal code

Reply:
I would guess we can stop all the workflows, deploy and resume them, which will trigger full resyncs for all the Help Centers but should be ok

Reply:
As far as I can tell the only worflow triggered by webhooks is stopIntercomSyncWorkflow -> Seems fine if we miss some

Reply:
this workflow is triggered by a webhook on `/uninstall`, it's clearly not fine at all if we miss some but this one is more easily reconstructible (we need the connectorId in the webhook and we're good to go)